gender_novels/novel.py

```diff
@@ -120,12 +121,13 @@ def __str__(self):
         return name
 
 
-    def _load_novel_text(self):
-        """Loads the text of a novel and removes boilerplate at the beginning and end
+    def load_novel_text(self):
+        """Loads the text of a novel and uses the remove_boilerplate_text() and
+        remove_table_of_contents() functions on the text of the novel to remove the boilerplate
+        text and table of contents from the novel. After these actions, the novel's text should be
+        only the actual text of the novel.
 
```

Comment:
This function needs a doctest.

Reply:
What would the doctest look like for a function like this, which processes an entire novel?

Reply:
Look at Doctest ellipsis for ways of making long tests shorter.  You can do:

>>> novel.text
It was a dark and stormy night...The End.
>>> len(novel.text)
892012